app/src/main/java/org/wikipedia/analytics/metricsplatform/MetricsPlatform.kt

```diff
@@ -0,0 +1,23 @@
+package org.wikipedia.analytics.metricsplatform
+
+import org.wikimedia.metrics_platform.MetricsClient
+import org.wikimedia.metrics_platform.context.AgentData
+import org.wikimedia.metrics_platform.context.ClientData
+import org.wikipedia.WikipediaApp
+
+object MetricsPlatform {
+    private val agentData: AgentData = AgentData(
+        WikipediaApp.instance.appInstallID,
+        "mobile app",
+        "android"
+    )
+
+    private val clientData: ClientData = ClientData(
+        agentData,
+        null,
+        null,
+        WikipediaApp.instance.wikiSite.authority()
+    )
+
+    val client: MetricsClient = MetricsClient.builder(clientData).build()
```

Comment:
Looks like we are building a new MP client for each event submitted. @dbrant how do you feel about having it built only once in the WikipediaApp class?

Reply:
I'm not seeing it built more than once here. Can you elaborate?

Reply:
My bad... i thought it was a get()